caffe2/serialize/inline_container.cc

```diff
@@ -740,8 +728,8 @@ void PyTorchStreamWriter::writeRecord(
       /*uncomp_size=*/0,
       /*uncomp_crc32=*/0,
       /*last_modified=*/nullptr,
-      /*user_extra_data=*/padding_.c_str(),
-      /*user_extra_data_len=*/padding_size,
```

Comment:
The old version does match the c++ code: https://github.com/richgel999/miniz/blob/0c30a001bc3c70770a8742ff00899e662f040c75/miniz_zip.c#L3266

Reply:
The new version matches the signature of ```mz_zip_writer_add_mem_ex_v2``` used in third-party's miniz 3.0.2.

Reply:
I think their header is not consistent with their c file actually haha
I guess either works.